Docs: Update time picker docs (#39847)
grafana/grafana: docs/sources/dashboards/time-range-controls.md

## Time units and relative ranges

The following time units are supported: `s (seconds)`, `m (minutes)`, `h (hours)`, `d (days)`, `w (weeks)`, `M (months)`, `Q (quarters)` and `y (years)`.

> The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/<time unit>` to the end. You can use `fQ (fiscal quarter)` and `fy (fiscal year)` to view fiscal periods.

Reviewer: ```suggestion
The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/<time unit>` to the end. To view fiscal periods, use `fQ (fiscal quarter)` and `fy (fiscal year)` time units.
```

Time settings are saved on a per-dashboard basis.

> **Timezone** and **fiscal year** settings can be changed from the time range controls by clicking the **Change time settings** button.
> 
> For more advanced time settings, click the **Dashboard settings** (gear) icon at the top of the screen. The settings are in the **Time Options** section of the General tab.

Reviewer: ```suggestion
You can change the **Timezone** and **fiscal year** settings from the time range controls by clicking the **Change time settings** button.

For more advanced time settings, click the **Dashboard settings** (gear) icon at the top of the UI. The advanced settings are in the **Time Options** section of the General tab.
```
